controllers/collections.js

```diff
@@ -57,23 +45,49 @@ exports.get = async function (req) {
     }
   }
 
-  const data = await req.db[req.params.collection].find(query, req.query.skip || req.query.offset,
-    req.query.limit, orderby, fields)
-  total = data.length
-  const limit = req.query.pageitems || (total > 10 ? 10 : total)
-  // calculate pagination in case `page`-queryarg was passed
-  const allowed = await Promise.all(data.map((doc) =>
-    util.notify('get', req, req.params.collection, doc).catch(() => false)
-  ))
-  let result = data.filter((doc, i) => allowed[i] === true)
-  if (req.query.page) {
+  if (req.query.page != null) {
+    req.query.page = parseInt(req.query.page)
     if (req.query.page <= 0) {
       throw new util.ApiError(400, 'invalid page number')
     }
-    result = util.createPagination(result, req.query.page, limit)
```

Comment:
nit: looks like this was the only usage of `createPagination`. Can you delete that as well?

Reply:
So i actually use it still in my project.. Wondering if other people may too.. I could always just port into my project, thoughts?

Reply:
Ahh, that makes sense! Let's keep it then.